fineract-provider/src/main/java/org/apache/fineract/infrastructure/reportmailingjob/service/ReportMailingJobWritePlatformServiceImpl.java

```diff
@@ -351,7 +352,7 @@ private DateTime createNextRecurringDateTime(final String recurrencePattern, fin
 
         // the recurrence pattern/rule cannot be empty
         if (StringUtils.isNotBlank(recurrencePattern) && startDateTime != null) {
-            final LocalDate nextDayLocalDate = startDateTime.plus(1).toLocalDate();
+            final LocalDate nextDayLocalDate = startDateTime.plus(Duration.millis(1)).toLocalDate();
```

Comment:
This looks suspicious/wrong..  if the variable is called next DAY, why add ONLY 1ms to Start? 

@ptuomola wanna help to code review this change for correctness?